main.pm

```diff
@@ -467,6 +467,7 @@ sub load_consoletests() {
         if (check_var("DESKTOP", "xfce")) {
             loadtest "console/xfce_gnome_deps.pm";
         }
+        loadtest "console/docker.pm";
```

Comment:
can you move this line to https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/main.pm#L463 just after mysql_srv test? I'd prefer test docker only if DESKTOP=textmomde for now like apache and mysql test, instead of running docker test a hundred times in every testjobs :)

the rest of code looks fine to me.